kivy/uix/recycleview/datamodel.py

```diff
@@ -0,0 +1,188 @@
+'''The data model part of the RecycleView model-view-controller pattern.
```

Comment:
Careful about the documentation, any file that need to be documentated must starts with a one line title followed by the ====, otherwise the doc can't get the title of the doc/file